soc/arm/microchip_mec/mec1501/Kconfig.defconfig.mec1501hsz

```diff
@@ -42,4 +42,11 @@ config I2C_XEC
 
 endif # I2C
 
+if ADC
+
+config ADC_XEC
+	def_bool y
```

Comment:
Why `def_bool` and not just `default`? This option is already defined as `bool` in [drivers/adc/Kconfig.xec](https://github.com/zephyrproject-rtos/zephyr/pull/19125/commits/2f8a9dcc1ede9160c6608d26b6da0ef896dccdcf#diff-3d9c6d68566610d3f10ac7a7d4ca58a4R9).

Reply:
Fixed.